apps/remix-ide/src/app/tabs/settings-tab.tsx

```diff
@@ -36,6 +36,7 @@ export default class SettingsTab extends ViewPlugin {
   }
   element: HTMLDivElement
   public useMatomoAnalytics: any
+  public useMatomoPerfAnalytics: any
```

Comment:
Is there any way we can add basic types and avoid any going forward? Any should be reserved for completely unresolvable situations.